### PR TITLE
Added files from Issues 1-5

### PR DESCRIPTION
**Pull request by ilchec**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "citation": "Chechuro 2021",
```
Comment (LinguList) [inline metadata.json]: Please provide the full title in the citation. Use Forthcoming, if data are not yet published.

**Review by LinguList — APPROVED** [review]
@ilchec, please remove teh .DS_STORE file. This is a mac-file that should not end up on github.

You also pushed another strange file ~$Russian... this should also be removed.

Otherwise, this is a good start! Please remove the binary files I mentioned, and then merge

Use git add file in the future, not git add *, this will avoid the problems!

**Comment by LinguList** [comment]
@ilchec, can you please respond to my requests and then merge this respository, so we can proceed?

**Comment by ilchec** [comment]
Sure, I'll do everything today in the evening or tomorrow morning!

**Comment by LinguList** [comment]
Okay, no real rush, I just wanted to make sure you saw my comments ;)

**Comment by ilchec** [comment]
Looks like everything's been fixed in the last update.

**Comment by ilchec** [comment]
@LinguList This is what I managed to do till 4 today. Can we make the initial Zenodo release to reserve the DOI and I'll keep working on the final version after the submission?

**Pull request merged**